examples/coronavirustwittermap/web/public/javascripts/map/pinmap.js

```diff
@@ -168,15 +167,18 @@ angular.module("cloudberry.map")
             }
             $scope.previousZoomLevel = event.level;
             sendPinmapTimeQuery();
+
+            $scope.deletePolygonLayers();
```

Comment:
I think these delete calls made us can not find the bug when they will be added again. Since we know where they are added, let's remove these delete calls. Only call the delete when we switch to pinmap from countmap.